docs/source/api.rst

```diff
@@ -17,8 +17,6 @@ API Reference
    api/inference
    api/glm
    api/gp
-   api/plots
-   api/stats
```

Comment:
I would probably add a line after the API toctree saying that PyMC3 delegates plots and stats to ArviZ and a link to https://arviz-devs.github.io/arviz/. Not sure why but even though pymc docs have only had links to ArviZ for quite some time now, I still see plenty of people using `pm.summary`, `pm.traceplot`... which may be confused by plots and stats sections disappearing from the docs.

Reply:
Sounds good @OriolAbril . How does this look?
I added a separate section, mentioning plotting and diagnostics.

<img width="313" alt="Screen Shot 2021-01-04 at 12 37 27 AM" src="https://user-images.githubusercontent.com/19514362/103517341-cd003a00-4e26-11eb-9fbd-efddbd6009bd.png">


Also, to the part of people using `pm.summary` or `pm.traceplot`, from what I understand, the initial dev for this repo didn't have arviz's advantages of being mature than what it is now. 

However, that is just paraphrasing what I read 🤷‍♂️  lol

Reply:
Also @OriolAbril , do you know which branch is the correct branch for the `v4` release?

Reply:
This looks great, thanks!

Reply:
I think it's named `v4`, right @twiecki ? So this PR should probably be based on this branch

Reply:
A `v4` branch (not the one we currently have in the repo) will be used for `v4` development, but we're not there yet.

I think we should merge this PR with `master` instead of trying to keep it in sync for so long.